src/index.js

```diff
@@ -1,37 +1,149 @@
 import "./style.css";
+
+class toDoTask {
+  constructor(desc, index, completed = false) {
+    this.desc = desc;
+    this.index = index;
+    this.completed = completed;
+  }
+}
+
 class UI {
   static displayTask() {
-    const task = [
-      {
-        desc: "buy fruits",
-        index: 1,
-        completed: false,
-      },
-      {
-        desc: "buy medicines",
-        index: 2,
-        completed: false,
-      },
-      {
-        desc: "buy household",
-        index: 3,
-        completed: false,
-      },
-    ];
-
-    task.forEach((works) => UI.addTasks(works));
+    // const task = [
+    //   {
+    //     desc: "buy fruits",
+    //     index: 1,
+    //     completed: false,
+    //   },
+    //
+    // ];
```

Comment:
- [ ] Please remove the following commented codes in order to keep your codes neat 😃

Reply:
done